graphql-code-generator/src/main/java/io/ballerina/graphql/generator/service/comparator/EnumMemberComparator.java

```diff
@@ -0,0 +1,32 @@
+package io.ballerina.graphql.generator.service.comparator;
+
+import io.ballerina.compiler.syntax.tree.EnumMemberNode;
+import io.ballerina.compiler.syntax.tree.Node;
+
+import static io.ballerina.graphql.generator.service.comparator.ComparatorUtils.getEnumMemberName;
+
+/**
+ * Utility class to store result comparing enum members.
+ */
+public class EnumMemberComparator {
+    private final Node prevMember;
+    private final Node nextMember;
+
+    public EnumMemberComparator(Node prevMember, Node nextMember) {
+        this.prevMember = prevMember;
+        this.nextMember = nextMember;
+    }
+
+    public boolean isMatch() {
```

Comment:
Multiple comparators with the `isMatch` and `generateCombinedResult()` methods are destined to be children of the same parent. ;)